src/xenia/gpu/xenos.h

```diff
@@ -1070,7 +1073,7 @@ XEPACKEDUNION(xe_gpu_texture_fetch_t, {
     uint32_t pitch : 9;                                  // +22 byte_pitch >> 5
     uint32_t tiled : 1;                                  // +31
 
-    TextureFormat format : 6;           // +0 dword_1
+    uint32_t format : 6;                // +0 dword_1, enum TextureFormat
```

Comment:
When a field has a type, it's preferred to use it instead for cleanliness — we're using types like Endian, MsaaSamples, TextureFormat wherever possible. It's already `: uint32_t`, it was fine that way.

Reply:
It's better to remove kUnknown from TextureFormat — it's only used in 4 places, 2 deprecated, 1 where it statically can't be obtained (and can be replaced with `? D24S8 : D24FS8` at the location of the call).